testnet-contracts/contracts/BankToken.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.0;
+
+import "../../node_modules/openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
+
+/**
+ * @title BankToken
+ * @dev Mintable ERC20 token controlled by bank contract
+ **/
+
+contract BankToken is ERC20Mintable {
```

Comment:
Do we want to use ERC20Detailed here too? And then just make name == _symbol and use the inherited constructor?